3ds/wslay/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+
+pkgname=3ds-wslay
+pkgver=1.1.1
+pkgrel=1
+pkgdesc='Wslay is a WebSocket library written in C.'
+arch=('any')
+url='https://github.com/tatsuhiro-t/wslay'
+license=('MIT')
+options=(!strip libtool staticlibs)
+groups=('3ds-portlibs')
+makedepends=('3ds-pkg-config' 'devkitpro-pkgbuild-helpers')
```

Comment:
devkitpro-pkgbuild-helpers has been superceded by dkp-toolchain-vars and several other packages that aren't needed for this library. Just use dkp-toolchain-vars here.